.env.development

```
@@ -0,0 +1,2 @@
+STAGE_SENTRY_DSN=https://758146f240094216826ede840ab34857@o7983.ingest.sentry.io/6703919
+STAGE_SENTRY_ENV=client-local-development
```

```suggestion
STAGE_SENTRY_ENV=local
```

these will all be for refinebio-web which is a client so we can just use local, preview, production